components/wb-bubble/bubble.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * @title WET-BOEW Bubble plugin container
+ * @overview Plugin used to open a overlay in the middle of the page
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @duboisp @cTopCanada @GormFrank @Ricokola
+ */
+( function( $, window, wb ) {
+"use strict";
+
+/*
+ * Variable and function definitions.
+ * These are global to the plugin - meaning that they will be initialized once per page,
+ * not once per instance of plugin on the page. So, this is a good place to define
+ * variables that are common to all instances of the plugin on a page.
+ */
+var componentName = "wb-bubble",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	defaults = {
+		i18nDict:
+		{
+			en: {
+				skip: "Skip to",
+				closeNotif: "Close notification"
+			},
+			fr: {
+				skip: "Passer à",
+				closeNotif: "Fermer la notification"
+			}
+		},
+		avatar: false
+	},
+	isNotif,
+	$document = wb.doc,
+
+	/**
+	 * Adjust bubble position to the footer of the current page
+	 * @method adjustBubbleMargin
+	 * @param {jQuery DOM element} $selector Element which is the actual bubble
+	 */
+	adjustBubbleMargin = function( $selector ) {
+
+		var $footer = $( "#wb-info" );
+
+		// Ensure that the bubble does not go passed the footer
+		if ( $footer.length ) {
+
+			// Add some white space over the footer for the bubble to sit
+			$footer.addClass( componentName + "-mrgn" );
```

Comment:
This margin class need to be added where the button is located in the page DOM. Not systematically to the site footer. This change is to support a focus order that match the visual sequence of the element rendered in the page.